x12-edi/2.5/antora.yml

```diff
@@ -1,6 +1,6 @@
 name: x12-edi-connector
-version: '2.1'
-display_version: 2.1 (Mule 4)
+version: '2.5'
+display_version: 2.5 (Mule 4)
 title: X12 EDI Connector
```

Comment:
Follow the template for title format.
I can't remember, but if the template uses the full connector name here, please do so.

Reply:
This is in the yaml file, so I don't want to change it.